pydantic_ai_slim/pydantic_ai/result.py

```diff
@@ -134,12 +78,27 @@ class RunResult(_BaseRunResult[ResultData]):
 
     data: ResultData
     """Data from the final response in the run."""
+    _result_tool_name: str | None
     _usage: Usage
 
     def usage(self) -> Usage:
         """Return the usage of the whole run."""
         return self._usage
 
+    def set_result_tool_return(self, return_content: str) -> None:
```

Comment:
Oh I see this was added in this PR, still, I don't really understand how it's meant to be used

Reply:
See my reply to your other comment.